### DISCUSSION TITLE
How do I change my spoken language? I know this repo is for programming language classification.

### DISCUSSION THREAD

**Discussion by kevinlu1248** [github-linguist/linguist#6501] [Q&A]
Our repo https://github.com/sweepai/sweep is not classified as English even though it's completely in English. Consequently, we don't show up in the English part of GitHub trending. I'm wondering where to change this setting since it's not documented anywhere and everywhere I search I end up here.

**Comment by lildude** [comment]
Please use the "Contact GutHub" link at the bottom of the page when viewing the repo to log a support ticket with GitHub's support.
